13/Dockerfile.c8s

```diff
@@ -44,7 +44,8 @@ COPY root/usr/libexec/fix-permissions /usr/libexec/fix-permissions
 RUN yum -y module enable postgresql:13 && \
     INSTALL_PKGS="rsync tar gettext bind-utils nss_wrapper postgresql-server postgresql-contrib" && \
     INSTALL_PKGS="$INSTALL_PKGS pgaudit" && \
-    yum -y --setopt=tsflags=nodocs install $INSTALL_PKGS && \
+    INSTALL_PKGS="$INSTALL_PKGS procps-ng util-linux postgresql-upgrade" && \
+    yum -y --setopt=tsflags=nodocs --setopt=install_weak_deps=false install $INSTALL_PKGS && \
```

Comment:
What difference in packages is there when using this option? I would rather not introduce it for already built image versions if the change is too disruptive.

Reply:
it not a must, it removes about 100MB of size - those are packages that are extra doesn't change / affect the core application

Reply:
For the existing images, I would choose the more defensive approach. I would just add the necessary packages.